docs/user/reference/yaml/pings.md

```diff
@@ -0,0 +1,118 @@
+# Pings YAML Registry Format
+
+Custom pings sent by an application or library are defined in YAML files which follow
+the [`pings.yaml` JSON schema](https://mozilla.github.io/glean_parser/pings-yaml.html).
+
+This files must be parsed by [`glean_parser`](https://pypi.org/project/glean-parser/) at build time
+in order to generate code in the target language (e.g. Kotlin, Swift, ...). The generated code is
+what becomes the public API to access the project's custom pings.
+
+For more information on how to introduce the `glean_parser` build step for a specific language /
+environment, refer to the ["Adding Glean to your project"](../../user/adding-glean-to-your-project/index.md)
+section of this book.
+
+{{#include ../../../shared/blockquote-info.html}}
+
+## Note on the naming of these files
+
+> Although we refer to pings definitions YAML files as `pings.yaml` throughout Glean documentation
+> this files may be named whatever makes the most sense for each project and may even be broken down
+> into multiple files, if necessary.
+>
+> This does not apply to Firefox Desktop, where all custom pings for a component must be declared
+> in a file called _exactly_ `pings.yaml`.
+
+## File structure
+
+```yaml
+# Schema
+$schema: moz://mozilla.org/schemas/glean/pings/2-0-0
+
+# Name
+search:
+  # Ping parameters
+  description: >
+    A ping to record search data.
+  include_client_id: false
+  notification_emails:
+    - CHANGE-ME@example.com
+  bugs:
+    - http://bugzilla.mozilla.org/123456789/
+  data_reviews:
+    - http://example.com/path/to/data-review
+```
+
+## Schema
+
+Declaring the schema at the top of a pings definitions file is advised,
+as it is what indicates that the current file is a pings definitions file.
+
+## Name
+
+Ping names are the top-level keys on pings definitions files. One single definition file
+may contain multiple ping declarations.
+
+Ping names are limited to lowercase letters from the [ISO basic Latin alphabet](https://en.wikipedia.org/wiki/ISO_basic_Latin_alphabet)
+and hyphens and a maximum of 30 characters.
+
+Ping names may not contain the words `custom` or `ping` in their names. These are considered redundant
+words in a custom ping declaration and will trigger a `REDUNDANT_PING` lint failure on `glean_parser`.
+
+["Capitalization"](../../user/metrics/adding-new-metrics.md#capitalization) rules apply to
+ping names on generated code.
+
+{{#include ../../../shared/blockquote-info.html}}
+
+### Reserved ping names
+
+> The names `baseline`, `metrics`, `events`, `deletion-request` and `all-pings` are reserved
+> and may not be used as the name of a custom ping.
```

Comment:
Should this mention that the words "custom" and "ping" are also, by default, reject?

Reply:
I added a note on that in the paragraphs above.